README.rst

```diff
@@ -64,8 +62,7 @@ list contains your navigation items, ready to output like in the examples above.
 
 Caching:
 --------
-To avoid hitting the database every time a user requests a page, the menu items are 
-cached if you have a cache configured. Caching is not used when ``settings.DEBUG`` is ``True``.
+To avoid hitting the database every time a user requests a page, the menu items are cached if you have a cache configured. Caching is not used when ``settings.DEBUG`` is ``True``.
```

Comment:
As above keeping line lengths short is a valid choice by the author and should stay as such unless there is a good reason for it.